### PR TITLE
feat(service-portal): Redirect to ids switch account screen 

### PR DESCRIPTION
**Pull request by disaerna**
# Service portal / User Menu

## What

* Remove delegation list from user menu and redirect to ids switch account screen instead. 
* Add "switch account" to user menu
* Add "switch account" to access control overview screen

Bonus: 
* Remove unnecessary comments
* Remove unused feature flag
 
## Why

Remove calls to Þjóðskrá and RSK api's on login, may cause future problems if traffic is high. Insead, only call when user is going to switch accounts -> redirect to the ids screen.

Solving current problem, however next fall we will code redesign of the whole access process.

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by thordurhhh — APPROVED** [review]
👍

File: libs/auth/react/src/lib/Authenticator/Authenticator.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ export const Authenticator: FC<Props> = ({ children, autoLogin = true }) => {
               prompt: 'none',
             }
           : {
-              login_hint: 'delegations',
+              prompt: 'select_account',
```
Comment (eirikurn) [inline libs/auth/react/src/lib/Authenticator/Authenticator.tsx:85]: 🙌

File: libs/shared/components/src/auth/UserMenu/UserDropdown.tsx [hunk 2]
```diff
@@ -55,15 +54,6 @@ export const UserDropdown = ({
   const actorName = actor?.name
   const isDelegationCompany = user.profile.subjectType === 'legalEntity'
 
-  const showDelegations =
-    useFeatureFlag(Features.delegationsEnabled, false).value || Boolean(actor)
```
Comment (eirikurn) [inline libs/shared/components/src/auth/UserMenu/UserDropdown.tsx:59]: Ah, we're ready to remove this feature flag?

**Review by eirikurn — APPROVED** [review]
Nice! We should probably change code owners of shared/components to hugsmidjan, sendiradid and/or island-ui.

**Pull request merged**